internal/gps/pkgtree/newLine.go

```diff
@@ -0,0 +1,103 @@
+package pkgtree
```

Comment:
need a copyright header (ugh, this should've caused our travis tests to fail...).

also, for consistency, filename should be snaky, not camel.

Reply:
I actually prefer snaky filenames, or even better with hyphens, but most other Go projects use camel case for file names. Is there something I don't know about, or just a convention localized to `dep`?

Reply:
it's the uniform approach in both stdlib and the toolchain, afaik. that's the convention we're aiming to follow.